src/components/cylc/gscan/GScan.vue

```diff
@@ -329,7 +329,7 @@ export default {
             { numeric: true, sensitivity: 'base' })
       })
         .map(workflow => {
-          return createWorkflowNode(workflow)
+          return createWorkflowHierarchyFromName(workflow)
```

Comment:
Note to self: maybe use just `createWorkflowNode`, but with a flag telling whether we want hierarchy or not? Or leave as-is…